Feature/add turkish language support 51

JCHeist opened the pull request.

JCHeist committed:
removed plists

JCHeist committed:
added turkish locale to Xcode project

JCHeist committed:
beginning of adding turkish locale

JCHeist committed:
about 1/3 the way done with running strings through google translate

JCHeist committed:
Some more strings work

JCHeist committed:
Going to fix one issue causing strings to be ouside quotes

JCHeist committed:
words all translated and errored formatting strings fixed formatting issues. Most of the comments are in turkish now though, not ideal?

JCHeist committed:
added turkish to supported languages in BartyCrouch enum

JCHeist committed:
brought back a few english comments, need to find another way to do this

JCHeist committed:
Swapped out comments in turkish for english ones

Because:
- needed englsih comments after i put the whole file in google translaete.

wrote a script for this. should have done that sooner.

JCHeist committed:
Added files that currently don't exist in workspace

Because:
- I think these files need to be source controlled. Especially the .lock and .resolved files. As the workspace get's bigger and we begin to use frameworks, having a source controlled xcode workspace will be necessarry in my opinion.

mosadialiou commented on loafwallet/tr.lproj/LaunchScreen.strings:
```diff
@@ -0,0 +1 @@
+
```
@JasperHeist empty file. must be deleted

kcw-grunt commented on loafwallet/src/Strings/tr.lproj/Localizable.strings:
```diff
@@ -232,1169 +232,1169 @@
 
-/* Fee Selector regular fee description */
-"FeeSelector.regularLabel" = "Estimated Delivery: 2.5 - 5+ minutes";
+/* Ücret Seçici normal ücret açıklaması */
```
@JasperHeist  Oh I see its confusing but the process is to leave the Comments in english (revert line 235) for example

kcw-grunt commented on loafwallet/src/Strings/tr.lproj/Localizable.strings:
```diff
@@ -232,1169 +232,1169 @@
 
-/* Fee Selector title */
-"FeeSelector.title" = "Processing Speed";
+/* Ücret Seçici başlığı */
```
@JasperHeist Oh I see its confusing but the process is to leave the Comments in english (revert line 238) for example. Line 239 is correct.  Meaning only the content on the right side of the label name.

kcw-grunt commented on loafwallet/src/Strings/tr.lproj/Localizable.strings:
```diff
@@ -232,1169 +232,1169 @@
 
-/* High fees error message */
-"Import.Error.highFees" = "Transaction fees would cost more than the funds available on this private key.";
+/* Yüksek ücret hata mesajı */
```
@JasperHeist same here and the rest of the file as commented above

kcw-grunt commented on Podfile.lock:
```diff
@@ -0,0 +1,129 @@
+PODS:
+  - BigInt (5.2.0)
```
Pods should be part of the .gitignore

kcw-grunt commented on TodayExtension/tr.lproj/MainInterface.strings:
```diff
@@ -1,14 +1,15 @@
 
 /* Class = "UILabel"; text = "send"; ObjectID = "QMM-JV-cTG"; */
-"QMM-JV-cTG.text" = "**send**";
+"QMM-JV-cTG.text" = "send";
```
@JasperHeist All changes to this file  should be reverted as we haven't supported localization for the Today Ext.  (someday)

kcw-grunt commented on loafwallet.xcodeproj/project.pbxproj:
```diff
@@ -4603,6 +4599,7 @@
 			children = (
 				75A2A79D1DA5934300A983D8 /* Base */,
 				249F976D236F86240045087A /* id */,
+				37198C1426549D6000F6F36C /* tr */,
```
looks good 😀

kcw-grunt commented on loafwallet.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
@JasperHeist This should be reverted and check the .gitignore.  Let's talk on making sure your env matches mine.

kcw-grunt commented on loafwallet.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
@JasperHeist This should be reverted and check the .gitignore.  Let's talk on making sure your env matches mine.

kcw-grunt reviewed (changes requested):
Some of these are going to needed to be reverted as the main changes should be the .strings  (TR) file and the added new localization.

mosadialiou reviewed (changes requested):
all IDE... files should be deleted since they are related to your Xcode environment.

mosadialiou commented on loafwallet.xcodeproj/xcuserdata/jcheist.xcuserdatad/xcschemes/xcschememanagement.plist:
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
@JasperHeist to be deleted. This file is related to your Xcode environment

mosadialiou commented on loafwallet.xcodeproj/project.xcworkspace/contents.xcworkspacedata:
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
@JasperHeist it looks like a file to delete. @kcw-grunt thoughts?

mosadialiou commented on loafwallet.xcworkspace/contents.xcworkspacedata:
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
@JasperHeist Looks like a file to remove from git. @kcw-grunt thoughts?

JCHeist committed:
Revert "Added files that currently don't exist in workspace"

This reverts commit 6db0c6087df6bab74d4cc0b622df35aa2839a030.

JCHeist committed:
Addressing PR

Because:
- A few strings files should not have been added. Deleted the unused ones xcode added

JCHeist committed:
Removed localizations for Main.storyboard and localizations for the TodayExtension

Because:
-  we do not currently support TodayExtension and the main.storyboard files were deleted since they were empty

JCHeist committed:
Today Extension.

Because:
- FIle was not saved during previous commit

kcw-grunt committed:
Clean up .gitignore

- Unignored Podfile.lock
- Added it back into source control
- Reformatted comments
- Included xcshareddata to source control

kcw-grunt committed:
Updated from  pod install

JCHeist committed:
What's different on Jasper's branch after a clean and build

kcw-grunt committed:
Re-add xcuserdata in gitignore

JCHeist committed:
Edited gitignore after fresh install of this branch. Removed reference to Main plist files that's not for helping devs.

Because:
- Edited .gitignore so the .xcodeproj and future .xcodeproj files will have their 'xcuserdata' folders ignored as well.

JCHeist committed:
Merge branch 'techdebt/jasper' into techdebt/60-jasper-diff
 -updated .gitignroe with changes

JCHeist committed:
deleted file in scuserdata

Because:
- before the gitignore is updated, this file needs to be removed so my old xcuserdata is not always in there

JCHeist committed:
readded gitignore changes

Because:
- to ignore all xcuserdata again

JCHeist committed:
removed xcuserdata from workspace that should be blown out now and will be ignored in the future

JCHeist committed:
.gitignore readded

Because:
- removed to fix extra file in source

JCHeist committed:
Updated comment in .gitignore

Because:
- For documentation

JCHeist committed:
added extra clause to gitignore for when xcuserdata folder is not named xcuserdata
Because:
- sometimes it is called [UserName].xcuserdata

JCHeist committed:
Corrected path to libbz2 module

Because:
- was red in the workspace

JCHeist committed:
Merge branch 'develop' into techdebt/60-jasper-diff

JCHeist committed:
Merge branch 'develop' into techdebt/project-for-dev-60-DevelopSync

JCHeist committed:
Merge branch 'techdebt/project-for-dev-60-DevelopSync' into techdebt/60-jasper-diff

JCHeist committed:
Merge branch 'techdebt/60-jasper-diff' into feature/Add-Turkish-Language-support-51

# Conflicts:
#	loafwallet.xcodeproj/project.pbxproj

Merged in dev env and develop changes

kcw-grunt reviewed (commented):
@JasperHeist Just let me know when you want me to review since we are now in sync.

JCHeist committed:
Merge commit '113e9fc734eb3e0ec6432ffaa4a09195223daaaf' into feature/Add-Turkish-Language-support-51

# Conflicts:
#	.gitignore
#	Podfile.lock

JCHeist committed:
removed zombie objects from scheme

JCHeist committed:
readed reference to main plist files

JCHeist committed:
removed launch screen.storyboard strings for indonesian language

JCHeist committed:
removed tribal changeling changes

kcw-grunt commented on loafwallet/src/Strings/tr.lproj/Localizable.strings:
```diff
@@ -0,0 +1,1382 @@
+/* About screen website label */
```
@JasperHeist how much fun did you have filling this out? Great? or Fantastic?

kcw-grunt commented on loafwallet/CHANGELOG.md:
```diff
@@ -1,9 +1,5 @@
 #  Changelog
 
```
Thanks for the cleanup

kcw-grunt commented on TodayExtension/Base.lproj/MainInterface.storyboard:
```diff
@@ -1,13 +1,9 @@
-    <device id="retina4_7" orientation="portrait">
-        <adaptation id="fullscreen"/>
-    </device>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="18122" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="M4Y-Lb-cyx">
```
Not sure if any adding Turkish should affect the Stpryboards at all.  Sorry for being cast. obvious but you could have accidentally `touched` the storyboard and that would register a change.  Will you check , unstage and / or explain my confusion?

JCHeist replied:
Not sure either. I know they changed when I first added localization because I had to unckeck the boxes on the storyboards... reverted. re-review requested

kcw-grunt reviewed (dismissed):
Just the storyboard issue. Other wise it looks good.  Just let me me know.

JCHeist committed:
Revert "Today Extension."

This reverts commit 97b68a2c0cd261f04547fd1723a31efbf11241e3.

JCHeist committed:
reverted changes to LItewallet storyboard

JCHeist committed:
changed storyboard part 2

kcw-grunt reviewed (approved):
LGTM
